pointers/pointers.c

```diff
@@ -21,7 +45,11 @@ void swap(int* a, int* b)
 */
 char *find_char(char *str, int c)
 {
-
+    if(str == 0);
+        printf("The character %c was not found.", %p);
+    else
+        printf("The character %c was found at position %d. \n", %p, str);
+    return 0;
```

Comment:
You can find a character in a string similar to how you found a node in a linked list before. All strings have a terminating null character (`'\0'`). You can iterate through a string, looking for the char and stop the iteration once you've found it or you've come to the end of the string.